plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetPreferences.java

```diff
@@ -91,6 +91,8 @@ public final class ResultSetPreferences {
     public static final String RESULT_TEXT_EXTRA_SPACES = "resultset.text.extra.spaces"; //$NON-NLS-1$
     public static final String RESULT_TEXT_LINE_NUMBER = "resultset.text.line.number";
 
+    public static final String RESULT_IMAGE_BROWSER = "resultset.image.browser";
```

Comment:
`result image browser`.. `image browser`, let's be more descriptive and call it at least `RESULT_IMAGE_USE_BROWSER_BASED_RENDERER` or something